.husky/pre-commit

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-$(npm bin)/lint-staged
+npx --no lint-staged
```

Comment:
we could use `npm exec` as well, but `npx` should be installed on all supported versions of node

Reply:
What kind of NPM do you have installed that does not have the `bin` subcommand? What does `npm --version` say if `npm bin` fails?

Reply:
It was removed in npm v9: https://github.com/npm/cli/releases/tag/v9.0.0

Reply:
Ah, OK, that seals the deal for me.